arch/risc-v/src/k230/Kconfig

```diff
@@ -22,22 +22,25 @@ config K230_RPTUN_SHM_SIZE
 	---help---
 		 RPTUN shared memory size in bytes.
 
-config K230_RPTUN_IPI
-	bool "use IPI for RPTUN"
-	default n
-	depends on K230_IPI
+config K230_RPTUN_IPI_DEV
+	int "IPI device num for RPTUN purpose"
+	default 0
 	---help---
-		Use Inter-Processor interruption to improve efficiency.
-		Not ready yet.
+		Choose the IPI device for RPTUN purpose, should be within 0..3
 
+endif # RPTUN
 
-config K230_RPTUN_WORK_DELAY
-	int "RPTUN work delay"
-	default 50
-	depends on !K230_RPTUN_IPI
-	---help---
-		 Ticks for polling RPTUN. Polling doesn't need IPI.
+config K230_IPI
```

Comment:
should we remove this Kconfig?

Reply:
I am unsure, as there might be other cases needing IPI. For example, someone might want use RPMsglite instead of RPTUN.